src/wp-includes/functions.wp-scripts.php

```diff
@@ -166,20 +167,23 @@ function wp_add_inline_script( $handle, $data, $position = 'after' ) {
  *                                    as a query string for cache busting purposes. If version is set to false, a version
  *                                    number is automatically added equal to current installed WordPress version.
  *                                    If set to null, no version is added.
- * @param bool             $in_footer Optional. Whether to enqueue the script before `</body>` instead of in the `<head>`.
- *                                    Default 'false'.
+ * @param array             $args     {
```

Comment:
this change will need an additional comment before merging to core indicating what the previous argument was and when it was changed.

Reply:
Good catch @adamsilverstein , do you have a good reference function I can use as a good example of this?

Reply:
@10upsimon, [this function](https://github.com/WordPress/wordpress-develop/blob/6.2/src/wp-includes/author-template.php#L344-L349) is an example of one that includes a second `@since` docbloc with a explanation of what changed in a later version than when the function was introduced.

Reply:
@adamsilverstein @joemcgill this has been addressed as part of a recent commit, I did the same for the `wp_enqueue_script` docblock. I've started to add `@since 6.3.0` as well and will continue to add for other functions that have been added.